Sources/RInAppMessaging/Router.swift

```diff
@@ -137,4 +234,92 @@ internal class Router: RouterType {
             return rootView
         }
     }
+
+    private func findParentViewForTooltip(_ sourceView: UIView) -> UIView {
+        let superview = sourceView.superview
+
+        if superview?.isKind(of: UIScrollView.self) == true {
+            return superview!
+        } else if superview?.isKind(of: UIWindow.self) == true {
+            if accessibilityCompatibleDisplay,
+               let transitionViewClass = NSClassFromString("UITransitionView"),
+               let transitionView = superview?.subviews.first(where: { !$0.isKind(of: transitionViewClass) }) {
```

Comment:
Is this logic safe to rely on continuing to work?

Reply:
It's been working so far for normal campaigns but it is a liability. There's no guarantee that Apple won't change view hierarchy in the next iOS version. I'll add this to tech debt log - there must be a better way to achieve the same result